--- config/FindVIGRANUMPY_DEPENDENCIES.cmake.orig	2025-03-09 14:56:01.000000000 -0400
+++ config/FindVIGRANUMPY_DEPENDENCIES.cmake	2025-03-09 14:57:02.000000000 -0400
@@ -38,7 +38,7 @@
             SET(BOOST_PYTHON_NAMES
                     boost_python-py${Python_VERSION_MAJOR}${Python_VERSION_MINOR}-mt
                     boost_python-${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}-mt
-                    boost_python${Python_VERSION_MAJOR}-mt
+                    boost_python${Python_VERSION_MAJOR}${Python_VERSION_MINOR}-mt
                     boost_python-mt)
         ENDIF()
         IF(Boost_LIB_SUFFIX)
